packages/atomic/src/components/common/facets/category-facet/children-as-tree-container.spec.ts

```diff
@@ -0,0 +1,105 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {html} from 'lit';
+import {describe, it, expect} from 'vitest';
+import {renderCategoryFacetChildrenAsTreeContainer} from './children-as-tree-container';
+
+interface CategoryFacetChildrenAsTreeContainerProps {
+  className?: string;
+}
+
+describe('renderCategoryFacetChildrenAsTreeContainer', () => {
+  const defaultProps: CategoryFacetChildrenAsTreeContainerProps = {
+    className: undefined,
+  };
+
+  const renderComponent = (
+    props: Partial<CategoryFacetChildrenAsTreeContainerProps> = {},
+    children = html`<li>Test child</li>`
+  ) => {
+    const mergedProps = {...defaultProps, ...props};
+    return renderFunctionFixture(
+      html`${renderCategoryFacetChildrenAsTreeContainer({props: mergedProps})(
+        children
+      )}`
+    );
+  };
+
+  it('should render a ul element with correct part attribute', async () => {
+    const container = await renderComponent();
+    const ul = container.querySelector('ul');
+
+    expect(ul).toBeInTheDocument();
+    expect(ul).toHaveAttribute('part', 'values');
+  });
+
+  it('should render children inside the ul element', async () => {
+    const container = await renderComponent(
+      {},
+      html`<li class="test-child">Test Item</li>`
+    );
+
+    const ul = container.querySelector('ul');
+    const child = container.querySelector('.test-child');
+
+    expect(ul).toBeInTheDocument();
+    expect(child).toBeInTheDocument();
+    expect(child).toHaveTextContent('Test Item');
+    expect(ul).toContainElement(child as HTMLElement);
+  });
+
+  it('should apply custom className when provided', async () => {
+    const container = await renderComponent({className: 'custom-class'});
+    const ul = container.querySelector('ul');
+
+    expect(ul).toHaveClass('custom-class');
+  });
+
+  it('should not apply class attribute when className is undefined', async () => {
+    const container = await renderComponent({className: undefined});
+    const ul = container.querySelector('ul');
+
+    expect(ul).not.toHaveAttribute('class');
+  });
+
+  it('should apply empty string as className', async () => {
+    const container = await renderComponent({className: ''});
+    const ul = container.querySelector('ul');
+
+    expect(ul).toHaveAttribute('class', '');
+  });
+
+  it('should render multiple children', async () => {
```

Comment:
Instead of having this as a separate test, we could modify the "should render children inside the ul element" test to make it render multiple children. I think we can safely presume that if the component can render multiple children, it can also render a single child 😁